flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/SlotManagerImplTest.java

```diff
@@ -976,6 +976,7 @@ public void testTaskManagerTimeoutDoesNotRemoveSlots() throws Exception {
 
 		try (final SlotManager slotManager = createSlotManagerBuilder()
 			.setTaskManagerTimeout(taskManagerTimeout)
+			.setRedundantSlotNum(0)
```

Comment:
Why do we need this? Isn't the default already 0?

Reply:
If default value is changed for certain organization or special situation, developers need to fix this test. Unit test should be run in any valid config.

Reply:
Make sense to me.